web/packages/teleport/src/Account/Account.tsx

```diff
@@ -156,122 +152,76 @@ export function Account({
-        : 'MFA method successfully deleted.';
-    addNotification('info', message);
-    onDeviceRemoved();
-  }
-
   return (
     <Relative>
-      <FeatureBox maxWidth={1440} margin="auto">
+      <FeatureBox margin="auto">
         <FeatureHeader>
+          <ArrowBack
+            mr={2}
+            size="large"
+            color="text.main"
+            onClick={() => history.goBack()}
+            style={{ cursor: 'pointer' }}
+          />
           <FeatureHeaderTitle>Account Settings</FeatureHeaderTitle>
         </FeatureHeader>
-        <Flex flexDirection="column" gap={4}>
-          <Box data-testid="passkey-list">
-            <AuthDeviceList
-              header={
-                <PasskeysHeader
-                  empty={passkeys.length === 0}
-                  passkeysEnabled={canAddPasskeys}
-                  disableAddPasskey={disableAddPasskey}
-                  fetchDevicesAttempt={fetchDevicesAttempt}
-                  onAddDevice={onAddDevice}
-                />
-              }
-              deviceTypeColumnName="Passkey Type"
-              devices={passkeys}
-              onRemove={onRemoveDevice}
-            />
-          </Box>
-          {!isSso && (
-            <PasswordBox
-              devices={devices}
-              passwordState={passwordState}
-              onPasswordChange={onPasswordChange}
-            />
-          )}
-          <Box data-testid="mfa-list">
-            <AuthDeviceList
-              header={
-                <Header
-                  title={
-                    <Flex gap={2} alignItems="center">
-                      Multi-factor Authentication
-                      <StatePill
-                        data-testid="mfa-state-pill"
-                        state={mfaPillState}
-                      />
-                    </Flex>
-                  }
-                  description="Provide secondary authentication when signing in
-                    with a password. Unlike passkeys, multi-factor methods do
-                    not enable passwordless sign-in."
-                  icon={<Icon.ShieldCheck />}
-                  showIndicator={fetchDevicesAttempt.status === 'processing'}
-                  actions={
-                    <ActionButtonSecondary
-                      disabled={disableAddMfa}
-                      title={
-                        disableAddMfa
-                          ? 'Multi-factor authentication is disabled'
-                          : ''
-                      }
-                      onClick={() => onAddDevice('mfa')}
-                    >
-                      <Icon.Add size={20} />
-                      Add MFA
-                    </ActionButtonSecondary>
-                  }
-                />
-              }
-              deviceTypeColumnName="MFA Type"
-              devices={mfaDevices}
-              onRemove={onRemoveDevice}
+        {!!errorMessage && <Danger dismissible>{errorMessage}</Danger>}
```

Comment:
what happens if someone dismisses this error?

Reply:
I marked it as dismissible so if a user understands the error and doesn't want a big red banner on their screen, they can get rid of it. It has no other action currently, but perhaps I should have it set errorMessage back to `null`. Is there anything else you would have it do?

Reply:
no, thats good. since there really isn't anything to "retry", i think getting rid of the banner is fine. thanks!